util-linux: take ${sbindir} from the environment if it is set there
<<<<<<< HEAD
fix the test, the [ ] syntax was getting eaten by autoconf

Signed-off-by: Phil Blundell <pb@pbcl.net>
Signed-off-by: Saul Wold <sgw@linux.intel.com
Upstream-Status: Inappropriate [configuration]

Index: util-linux-2.22.1/configure.ac
===================================================================
--- util-linux-2.22.1.orig/configure.ac
+++ util-linux-2.22.1/configure.ac
@@ -73,7 +73,10 @@ AC_SUBST([localstatedir])
=======

Signed-off-by: Phil Blundell <pb@pbcl.net>
Upstream-Status: Inappropriate [configuration]

--- util-linux-2.21.2/configure.ac~	2012-09-22 09:29:12.354215377 +0100
+++ util-linux-2.21.2/configure.ac	2012-09-22 16:47:14.713628946 +0100
@@ -67,7 +67,9 @@
>>>>>>> cb9658cf8ab6cf009030dcadde9dc6c54b72bddc
 usrbin_execdir='${exec_prefix}/bin'
 AC_SUBST([usrbin_execdir])
 
-usrsbin_execdir='${exec_prefix}/sbin'
<<<<<<< HEAD
+if test -z "$usrsbin_execdir" ;
+then
=======
+if [ -z "$usrsbin_execdir" ]; then
>>>>>>> cb9658cf8ab6cf009030dcadde9dc6c54b72bddc
+   usrsbin_execdir='${exec_prefix}/sbin'
+fi
 AC_SUBST([usrsbin_execdir])
 
<<<<<<< HEAD
 case $libdir in
=======
 usrlib_execdir='${exec_prefix}/'$libdirname
>>>>>>> cb9658cf8ab6cf009030dcadde9dc6c54b72bddc
